src/cls/tabular/fbwriter_fbu.cc

```diff
@@ -0,0 +1,932 @@
+        auto float_vect_fb = builder.CreateVector( float_vect ) ;
+        auto data = Tables::CreateSDT_FLOAT_FBU( builder, float_vect_fb ) ;
+        auto col = Tables::CreateCol_FBU(
+          builder,                              //builder
+          rids_vect_fb,                         //rids
+          nullbits_vector_fb,                   //nullbits
+          col_name,                             //col_name
+          col_index,                            //col_index
+          Tables::DataTypes_FBU_SDT_FLOAT_FBU,  //data_type
+          data.Union() ) ;                      //data
+        cols_vect.push_back( col ) ;
+      }
+      else if( boost::algorithm::ends_with( key, std::to_string( Tables::SDT_STRING ) ) ) {
+        std::vector< flatbuffers::Offset<flatbuffers::String> > string_vect = filedata.listof_string_vect_raw[ std::stoi(index) ] ;
+        auto string_vect_fb = builder.CreateVector( string_vect ) ;
+        auto data = Tables::CreateSDT_STRING_FBU( builder, string_vect_fb ) ;
+        auto col = Tables::CreateCol_FBU(
+          builder,                              //builder
+          rids_vect_fb,                         //rids
+          nullbits_vector_fb,                   //nullbits
+          col_name,                             //col_name
+          col_index,                            //col_index
+          Tables::DataTypes_FBU_SDT_STRING_FBU, //data_type
+          data.Union() ) ;                      //data
+        cols_vect.push_back( col ) ;
+      }
+      else {
+        std::cout << ">>> unrecognized key '" << key << "'" << std::endl ;
+        exit(1) ;
+      }
+
+      //save the Cols flatbuffer
+      if( inputs.cols_per_fb == 1 ||
+          ( (i+1) % inputs.cols_per_fb == 0 ) ||
+          ( (i+1) == ncols ) ) {
+
+        if( debug )
+          std::cout << "saving bl to bl_seq" << std::endl ;
+
+        auto cols_data = builder.CreateVector( cols_vect ) ;
+        std::vector< flatbuffers::Offset< Tables::Col_FBU > > empty_cols_vect ;
+        cols_vect = empty_cols_vect ; //empty out collection vector
+
+        auto cols = CreateCols_FBU(
+          builder,
+          cols_data ) ;
+
+        std::vector< uint8_t > delete_vector ( nrows, 0 ) ; //initialize with one 0 per row.
+        auto delete_vector_fb = builder.CreateVector( delete_vector ) ;
+
+        auto root = CreateRoot_FBU(
+          builder,                          // builder
+          SFT_FLATBUF_UNION_COL,           // data_format_type
+          0,                                // skyhook_version
+          0,                                // data_structure_version
+          0,                                // data_schema_version
+          schema_string_fb,                 // data_schema string
+          db_schema_name,                   // db_schema_name TODO: parameterize
+          (uint32_t)nrows,                  // nrows
```

Comment:
preferably change to static_cast whenever possible

Reply:
tried changing each of the six reinterpret_cast calls into static_cast, but all failed.

Reply:
ok thanks, the reinterpret cast cannot be changed, i just meant to replace any c-style casts (new_type) with static_cast for clarity and type safety.